src/synthesizer/abundances/elements.py

```diff
@@ -0,0 +1,137 @@
+
+class Elements:
```

Comment:
As above: not sure why this is a class and why it's inherited.

Reply:
Changed